cfp.html

```diff
@@ -36,10 +40,9 @@ <h1 style="margin-top: 50px" class="section-header align-self-start">Speaker ass
             <p>If you’re a first time speaker and thinking EuRuKo is out of your reach, you’re mistaken! We encourage first-time speakers to use EuRuKo as a springboard for a career of awesome talks!</p>
             <p>Ruby’s in the name, so Ruby should be the game! Rails talks are cool, of course, but the content of the conference is framework-agnostic. And hey, if you’ve got a non-Ruby or ‘for Rubyists’ talk in mind, send it our way!</p>
             <h1 style="margin-top: 50px" class="section-header align-self-start">The perks</h1>
-            <p>What do you get as a speaker, I hear you 🤔’ing? If you’re selected as a speaker, here’s what we can offer you:</p>
+            <p>What do you get as a speaker, I hear you 🤔’ing? Aside from having free entry to the conference (which really barely counts as a perk!), if you’re selected as a speaker, here’s what we can offer you:</p>
```

Comment:
Not sure if this emoji is descriptive enough, at least when I first read, I had to spend extra second to understand the word. Maybe simple text is better? Not sure how does this reflect on the website. What others think?

Reply:
You're right. I was trying to be cute, but I think it's a bit obscure. Thank you for the feedback. I'll change it